### ISSUE TITLE
implement version subcommand

### ISSUE THREAD

**Issue by nhooyr** [google/gvisor#175]
When I set the default runtime for docker v18.09.4-rc1 to runsc, I see this in the logs repeating continuously:

```
time="2019-03-28T16:09:18.494562592Z" level=warning msg="failed to retrieve /usr/local/bin/runsc version: exit status 2"
time="2019-03-28T16:09:21.185047223Z" level=warning msg="failed to retrieve /usr/local/bin/runsc version: exit status 2"
time="2019-03-28T16:09:22.330433216Z" level=warning msg="failed to retrieve /usr/local/bin/runsc version: exit status 2"
time="2019-03-28T16:09:36.461863659Z" level=warning msg="failed to retrieve /usr/local/bin/runsc version: exit status 2"
```

Would be nice if there was a `version` subcommand to satisfy docker.

**Comment by amscanne** [comment]
Looking at this now. Interesting that Docker seems to parse the version then ignore it in favor of the commit, which it records for later.

https://github.com/docker/docker-ce/blob/fec57a028b07018ac0539687c8138cfca9da7943/components/engine/daemon/info_unix.go#L37

There may be other assumptions that it makes based on that. Either way, I don't see any harm in supporting a --version flag, since it will otherwise set the commit to "N/A".

**Comment by amscanne** [comment]
Fixed in 7543e9e.
